### ISSUE TITLE
Testmo upload does not include describe group  words from tests

### ISSUE THREAD

**Issue by greggjensencart** [jonasclaes/playwright-testmo-reporter#2]
This is a nice reporter; I like how it maps the steps.  One issue currently, is the reporter does not include the `describe` group words from tests.

`products/add-product.spec.ts`
```TypeScript
test.describe('add product from scratch', () => {

    test('adds single', async ({ page }) => {
          ....
    });
});
```

## playwright-testmo-reporter
![image](https://github.com/jonasclaes/playwright-testmo-reporter/assets/106989443/e98753ba-082f-4a9d-b544-38f3ce05876c)

## junit-reporter
![image](https://github.com/jonasclaes/playwright-testmo-reporter/assets/106989443/59d29fcf-e031-4f5a-9c4b-bb87ec687ae4)

**Comment by jonasclaes** [comment]
Hello @greggjensencart!
Thank you for creating this issue!

I can see why this feature would be useful.
Would you like to implement this yourself and create a PR? If not, then I'd be happy to implement this.

**Comment by greggjensencart** [comment]
It would take me a bit to get familiar with the code base.  If you can add it, that would be great.

**Comment by jonasclaes** [comment]
Sure thing! I'll get started on that!

**Comment by jonasclaes** [comment]
Woops, closed that too quickly.

I have released a new version, `v1.2.0`.
For reference, you can find the packaged version here: https://www.npmjs.com/package/playwright-testmo-reporter/v/1.2.0

I have verified it locally as working. Please refer to the README to learn about the new testTitleDepth config attribute.

Let me know if it works! And if it does, you can close this issue! 😄

**Comment by jonasclaes** [comment]
Closing this issue due to inactivity.
